### ISSUE TITLE
Fastplace: dropdowns should be disabled for pre-defined building assignments

### ISSUE THREAD

**Issue by RHolton** [widelands/widelands#5573]
For those pre-defined building slots in the Fastplace UI, there are still dropdown menus that can be changed to other than the pre-defined building. Changing them appears to have no impact on the actual assignment. They only serve to confuse the user.

These dropdowns should be disabled, or some other change to the UI should be made to make it clear that the actual key assignment will always be to the pre-defined building.

Using Version 1.1 ~ git1(ale798f@master) Debug.

**Comment by Noordfrees** [comment]
The predefined entries can be changed just like all others.

> Changing them appears to have no impact on the actual assignment.
 
If that's true then this is a bug and must be fixed. However I could not reproduce that behaviour.
